app/services/notes_grouper.rb

```diff
@@ -64,14 +64,10 @@ def group
                                end
 
         when 0
-          # If there is no matching note number, increment the
-          # current highest note number and use it for note
-          # elements for this type
-          max_note = ordered_mapping.max_by { |k, _v| k[/\d+/].to_i }.first
```

Comment:
This code seemed to only get used in the situation of no displayLabels or types and was the cause of the new notes columns getting created for notes that didn't have labels or types. It may be I'm not understanding something about the grouping logic, however. 

I left the `when 0` statement, even though the action is now the same as with the `else` statement, although I could remove this. Was thinking it might be helpful to have the `0` scenario not mixed with the `>1` scenario for clarity with this complicated logic, but open to removing it and just adding a comment.

Reply:
Oooh, if this isn't needed, that'd be great.

Reply:
Have dropped this.